core/src/com/unciv/ui/tilegroups/CityButton.kt

```diff
@@ -237,6 +239,17 @@ class CityButton(val city: CityInfo, private val tileGroup: WorldTileGroup): Tab
         iconTable.addActor(cityStrengthLabel)        // We create this here to we can .toBack() it as well.
         cityStrengthLabel.toBack()
 
+        if (city.civInfo.isCityState()) {
+            val cityStateImage = when (city.civInfo.cityStateType) {
```

Comment:
`val cityStateImage = ImageGetter.getImage("OtherIcons/${city.civInfo.cityStateType.name}").apply { color = secondaryColor }` would hold so far... And reusing a texture region under a different name is done by simply giving the packer a bit-identical copy. In case you would like that better.

Reply:
Yep, that's better, now it's future-proof for Religious city-states.